src/App.tsx

```diff
@@ -34,170 +92,125 @@ export const App: React.FC = () => {
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
+          <div>
+            {todos &&
+              getFormedTodos(todos).map((item, index) => {
+                return (
+                  <div
+                    data-cy="Todo"
+                    key={'id: ' + index}
+                    className={classNames('todo', {
+                      completed: item.completed,
+                    })}
+                  >
+                    <label className="todo__status-label">
+                      <input
+                        data-cy="TodoStatus"
+                        type="checkbox"
+                        checked={item.completed}
+                        className="todo__status"
+                      />
+                    </label>
+
+                    <span data-cy="TodoTitle" className="todo__title">
+                      {item.title}
+                    </span>
+
+                    <button
+                      type="button"
+                      className="todo__remove"
+                      data-cy="TodoDelete"
+                    >
+                      ×
+                    </button>
+
+                    <div data-cy="TodoLoader" className="modal overlay">
+                      <div className="modal-background has-background-white-ter" />
+                      <div className="loader" />
+                    </div>
+                  </div>
+                );
+              })}
           </div>
         </section>
 
-        {/* Hide the footer if there are no todos */}
-        <footer className="todoapp__footer" data-cy="Footer">
-          <span className="todo-count" data-cy="TodosCounter">
-            3 items left
-          </span>
-
-          {/* Active link should have the 'selected' class */}
-          <nav className="filter" data-cy="Filter">
-            <a
-              href="#/"
-              className="filter__link selected"
-              data-cy="FilterLinkAll"
-            >
-              All
-            </a>
-
-            <a
-              href="#/active"
-              className="filter__link"
-              data-cy="FilterLinkActive"
-            >
-              Active
-            </a>
+        {todos.length > 0 && (
+          <footer className="todoapp__footer" data-cy="Footer">
+            <span className="todo-count" data-cy="TodosCounter">
+              {todos.filter(x => !x.completed).length} items left
+            </span>
 
-            <a
-              href="#/completed"
-              className="filter__link"
-              data-cy="FilterLinkCompleted"
+            {/* Active link should have the 'selected' class */}
+            <nav className="filter" data-cy="Filter">
+              <a
+                href="#/"
+                className={classNames('filter__link', {
+                  selected: filter === null,
```

Comment:
```suggestion
                  selected: !filter,
```

Reply:
Filter may be false there

Reply:
yep, and it's not the best solution - that `filter ` stores `boolean | null`. it'd be way easier to understand the logic if the filter state stores `Filter` enum and you just have `switch` block in the function that filters todos

So I recommend you to change it in the next stage